selfdrive/ui/translations/main_fr.ts

```diff
@@ -264,11 +264,11 @@
     </message>
     <message>
         <source>Pair Device</source>
-        <translation type="unfinished"></translation>
+        <translation>Associez l&apos;appareil</translation>
     </message>
     <message>
         <source>PAIR</source>
-        <translation type="unfinished"></translation>
+        <translation>Assoiez</translation>
```

Comment:
Associez?

Reply:
First I thought "Coupler" but I saw that "Associer" is the verb used in all other instances ("coupler" never appears). I wanted the new translations to be consistent with the text. What do you think? should we change those other instances to "Coupler"? Or another verb? Also maybe we should put this one in all caps right... Let me change that.